packages/lib-user/package.json

```diff
@@ -26,7 +27,7 @@
     "dev": "BABEL_ENV=es6 webpack serve --config webpack.dev.js",
     "eslint": "eslint ./src --fix | snazzy",
     "prepare": "yarn build",
-    "test": "BABEL_ENV=es6 mocha --loader=@node-loader/babel --config ./test/.mocharc.json \"./src/**/*.spec.js\"",
+    "test": "mocha --config ./test/.mocharc.json ./.storybook/specConfig.js \"./src/**/*.spec.js\"",
     "test:ci": "echo \"No CI tests yet...\" && exit 0",
```

Comment:
I could add lib-user tests to CI (`test:ci`) now, could also `build-storybook`? Not necessary, but for consideration.

Reply:
Totally up to you! There's only one UI component so far, so I'm not worried if you want to wait to incorporate CI until more of the app is built.